.eslintrc.json

```diff
@@ -91,7 +89,7 @@
         "no-eval": "error",
         "no-new-wrappers": "error",
         "no-prototype-builtins": "off",
-        "no-shadow": "error",
+        "no-shadow": "off",
```

Comment:
Do we want to turn off no-shadow -- seems fairly useful?

Reply:
`no-shadow` started complaining about every enum definition being shadowed after updating eslint packages.

Reply:
Thanks for the explanation. I thought it was something like that.